deny.toml

```diff
@@ -23,6 +23,8 @@ skip-tree = [
     { name = "windows-sys" },
     # old version pulled in by rustls via ring
     { name = "spin" },
+    # lots still pulls in syn 1.x
+    { name = "syn" },
```

Comment:
I tried to remove duplicate syn versions but still lots of crates that need to be updated. I'm gonna submit some patches but don't it should block us from updating.